static/app/views/dashboardsV2/detail.tsx

```diff
@@ -416,6 +411,15 @@ class DashboardDetail extends Component<Props, State> {
         filters: {...newModifiedDashboard.filters, ...activeFilters},
       },
     });
+    if (!isEqual(activeFilters, dashboard.filters?.release)) {
+      browserHistory.replace({
```

Comment:
Should we do browserHistory.push instead so that we can go back on previous dashboard filter states 👀

Reply:
Great idea! Thanks for the suggestion